server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationIT.java

```diff
@@ -112,6 +113,11 @@
 @OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.TEST, numDataNodes = 0)
 public class SegmentReplicationIT extends SegmentReplicationBaseIT {
 
+    @Override
+    protected Settings featureFlagSettings() {
+        return Settings.builder().put(super.featureFlagSettings()).put(FeatureFlags.ZSTD_COMPRESSION, "true").build();
```

Comment:
😕   As a follow up (not for 2.9.1 patch release) we need to tease zstd testing out as a separate segrep test.

Reply:
makes sense, created an issue https://github.com/opensearch-project/OpenSearch/issues/9488 so that this recommendation is not lost.